configure.ac

```diff
@@ -4787,6 +4827,27 @@ jit_status="Building and using the JIT"
 libsuffix=".so"
 
 case "$host" in
+     *-*-aix*)
+	dnl on AIX/PASE, shared libraries can be and usually are inside archives
+	dnl so, we specify them by libfoo.a(libfoo.so.0) for libtool's conventions,
+	dnl or libfoo.a(shr[_64].o) for the AIX system convention,
+	dnl or lib.so[.0](shr[_64].o) for libtool's hybrid convention
+	dnl we may hardcode 64-bit names at times, but we don't do 32-bit AIX
+	LIBC="libc.a(shr_64.o)"
+	# This is particularily nasty because we need to refer to libintl by version, which may not necessarily match what a system has.
+	INTL="libintl.a(libintl.so.8)"
```

Comment:
Can any of the behaviors or versions be detected instead of relatively hard coding?

Reply:
I could try something based on parsing the output of ar, but that seems somewhat flaky. I don't think autotools has a readymade test for this kind of thing.

Reply:
It seems like it should be easier, but probably anything is ok.